connector/http-agent/app/express/interceptors/deviceIdentification.js

```diff
@@ -29,9 +30,9 @@ module.exports = ({ cache, config }) => ({
           [req.tenant, req.deviceId] = messageKey;
           return next();
         } catch (e) {
-          err.message =
+          Forbidden.message =
             'Error trying to get tenant and deviceId in CN of certificate.';
```

Comment:
Hm... a 403 error is caused by invalid permissions, so your error message should mention that and signalize the user.
You got it right in: Forbidden.message = 'Client certificate is invalid';